assignments/arrays.js

```diff
@@ -75,15 +75,10 @@ console.log(`Make:`+ " " + inventory[lastCar.length-1].car_make + " "+ `Model:`
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-for (let i = 0; i < inventory.length; i++){
-    let model = inventory[i].car_model;
-    carModels.push(model);
-    carModels.sort();
-    
-    }
-
-    console.log(carModels);
-
+for (let i = 0; i < inventory.length; i++) {
+  carModels.push(inventory[i].car_model);
+}
+console.log(carModels.sort());
```

Comment:
awesome I see you made those changes here and placing the sort outside of the for loop.